dev-libs/wlroots/metadata.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>speedjack95@gmail.com</email>
+		<name>Niccolò Scatena</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Pluggable, composable, unopinionated modules for building a Wayland
+		compositor.
+
+		wlroots implements a huge variety of Wayland compositor features and
+		implements them right. By using wlroots, you get high performance,
+		excellent hardware compatibility, broad support for many wayland
+		interfaces, and comfortable development tools.
+
+		wlroots is developed under the direction of the <pkg>dev-libs/sway</pkg> project.
+	</longdescription>
+	<use>
+		<flag name="png">Enable examples that use libpng (PNG images)</flag>
```

Comment:
png use flag is global. no need to replicate it here.

Reply:
Huh, did you change what it said? I was trying to make this comment
before github crapped the bed and I swear it was word-for-word the
same as the global use.desc file's description.

Reply:
No, I didn't change it :P